### PR TITLE
Fix/docs

### PR DESCRIPTION
**Pull request by CrepeGoat**
Corresponds to MideTechnology/endaq-python#6.

### CODE REVIEW

File: endaq/ide/info.py [hunk 1]
```diff
@@ -16,22 +16,34 @@
 from .measurement import ANY, get_channels
 
 
+__all__ = [
+    "parse_time",
+    "format_channel_id",
+    "format_timedelta",
+    "format_timestamp",
```
Comment (StokesMIDE) [inline endaq/ide/info.py:23]: These are primarily for internal use and shouldn't be in `__all__`. If, in an edge-case, a user needs them, they can be imported explicitly.

File: endaq/ide/files.py [hunk 2]
```diff
@@ -91,27 +91,28 @@ def get_doc(name=None, filename=None, url=None, parsed=True, start=0, end=None,
...
-            * ``"3:22:11"`` (3 hours, 22 minutes, 11 seconds)
-            * ``"1d 3:22:11"`` (1 day, 3 hours, 22 minutes, 11 seconds)
-        * `datetime.timedelta` or `pandas.Timedelta` (time from the
-          recording start)
-        * `datetime.datetime` (an explicit UTC time)
+
+    * `int`/`float` (Microseconds from the recording start)
+    * `str` (formatted as a time from the recording start, e.g., `MM:SS`,
+      `HH:MM:SS`, `DDd HH:MM:SS`). More examples:
+
```
Comment (StokesMIDE) [inline endaq/ide/files.py:108]: Why the line break here? If it is necessary, why not after the indented section as well?

**Review by StokesMIDE — CHANGES_REQUESTED** [review]
One comment, one minor change request.

File: endaq/ide/files.py (hunk 2 above)
Comment (CrepeGoat) [reply endaq/ide/files.py:108]: My goal was to make the minimal number of changes required to remove build warnings; the line break there was necessary, and the others weren't.

Having said that, I don't care too much about how these are formatted. So if you'd like to add in matching line breaks for the indented sections, feel free to do so. But I'd ask that you make the changes yourself if you don't mind; I think doing so will avoid a lot of back and forth conversation between us.

File: endaq/ide/info.py (hunk 1 above)
Comment (CrepeGoat) [reply endaq/ide/info.py:23]: can you tell me which ones exactly are intended for "internal use"? I can't tell the difference. (related note: should the "internal use" functions have names prefixed with underscores?)

Comment (StokesMIDE) [reply endaq/ide/info.py:23]: > can you tell me which ones exactly are intended for "internal use"? I can't tell the difference.

The four highlighted. 

> (related note: should the "internal use" functions have names prefixed with underscores?)

By convention, the underscore implies a function shouldn't be used. These aren't *exclusively* for internal use; they might be useful to some, but most users won't care. I'd prefer they were excluded from `__all__`, but honestly, it really doesn't matter too much.

File: endaq/ide/files.py (hunk 2 above)
Comment (StokesMIDE) [reply endaq/ide/files.py:108]: Yeah, it isn't really important. I'm admittedly a bit particular about formatting. It wasn't meant as a request for change.

File: endaq/ide/info.py (hunk 1 above)
Comment (CrepeGoat) [reply endaq/ide/info.py]: it makes no difference to me so I don't mind. addressed in 3772262ad285ab6ed6cd9b45c5c3bba7ffdbaca0

**Review by StokesMIDE — APPROVED** [review]

**Pull request merged**